src/trading/TradingEscapeHatch.sol

```diff
@@ -87,7 +82,7 @@ contract TradingEscapeHatch is DelegationTarget, ITradingEscapeHatch {
             // FIXME: Think about this math, can shiftedPrice * range be greater than 2*254?
             //        Can shiftedPrice / _sumOfBids lead to rounding errors?  Should we * then / or / then *?
             //        Yes it can be greater, shifted / _sumOfBids will lead to rounding errors
-            frozenShareValues[_market][_tempOutcome] = _shiftedPrices[_tempOutcome].fxpMul(_range, 1 ether).fxpDiv(_sumOfBids, 1 ether);
+            frozenShareValues[_market][_tempOutcome] = _shiftedPrices[_tempOutcome].fxpMul(_market.getMarketDenominator(), 1 ether).fxpDiv(_sumOfBids, 1 ether);
```

Comment:
Is it intentional that this still does a `fxpMul` with a base of `1 ether`?  I thought with the changes in this PR we didn't have any more 1 ether base fixed point math being done?

Reply:
Yea I missed this. Will change